src/libostree/ostree-sysroot-deploy.c

```diff
@@ -2082,6 +2082,44 @@ install_deployment_kernel (OstreeSysroot   *sysroot,
   g_autofree char *options_key = ostree_kernel_args_to_string (kargs);
   ostree_bootconfig_parser_set (bootconfig, "options", options_key);
 
+  g_autoptr(GError) local_error = NULL;
+  GKeyFile *config = ostree_repo_get_config (repo);
+  gchar **read_values = g_key_file_get_string_list (config, "sysroot", "bls-append-except-default", NULL, &local_error);
+  /* We can ignore not found errors */
+  gboolean not_found = g_error_matches (local_error, G_KEY_FILE_ERROR, G_KEY_FILE_ERROR_KEY_NOT_FOUND) || \
```

Comment:
This is fine as is (I looked and it seems `g_error_matches` checks for `error == NULL`, but it's more idiomatic to check `if (read_values == NULL)` first.

IOW (sorry whitespace damage, typing in GH comments):

```
if (!read_values)
  {
 gboolean not_found = g_error_matches (local_error, G_KEY_FILE_ERROR, G_KEY_FILE_ERROR_KEY_NOT_FOUND) || \
                       g_error_matches (local_error, G_KEY_FILE_ERROR, G_KEY_FILE_ERROR_GROUP_NOT_FOUND);
  if (not_found)
    {
      g_clear_error (&local_error);
    }
  else
   {
      g_propagate_error (error, g_steal_pointer (&local_error));
      return FALSE;
   }
  }
```